### ISSUE TITLE
`trepan3k` unusable on Python 3.9.19 because of an import error

### ISSUE THREAD

**Issue by nejmeijer** [Trepan-Debuggers/python3-trepan#71]
Installing `trepan3k` with `pip` according the the [RTD instructions](https://python3-trepan.readthedocs.io/en/latest/install.html) gives a command that when invoked crashes with
> ModuleNotFoundError: No module named 'tracer.tracer'

like described in rocky/pytracer#8. @rocky's comment these seems to indicate that – until the next `trepan3k` release – the work-around is to install from source (which I assume refers to the commands from `git clone` onwards on the [install page](https://python3-trepan.readthedocs.io/en/latest/install.html#using-easy-install)). This still gives the same error.

The other work-around – downgrading `pytracer` to 0.3.2 – seems forbidden by `pip` (assuming `trepan3k` would even work with `pytracer<2.0.0`).

Installing from a [wheel](https://github.com/Trepan-Debuggers/python3-trepan/releases/download/1.3.0/trepan3k-1.3.0-39-none-any.whl) is not supported on my platform, so I did not test that.

I am raising this issue in addition to rocky/pytracer#8 because `trepan3k` is unusable in the current state, but also to ask @rocky to share the deeper understanding he seems to have of this error.

**Comment by rocky** [comment]
> Installing `trepan3k` with `pip` according the the [RTD instructions](https://python3-trepan.readthedocs.io/en/latest/install.html) gives a command that when invoked crashes with
> 
> > ModuleNotFoundError: No module named 'tracer.tracer'

There is too little information here to understand what you did and what is wrong.  

What version of trepan3k are you running? 

> 
> like described in [rocky/pytracer#8](https://github.com/rocky/pytracer/issues/8). @rocky's comment these seems to indicate that – until the next `trepan3k` release –

A release came out end of last November.

>  the work-around is to install from source (which I assume refers to the commands from `git clone` onwards on the [install page](https://python3-trepan.readthedocs.io/en/latest/install.html#using-easy-install)). This still gives the same error.

I possibly have encountered something similar. Make sure to _uninstall_ trepan3k before installing a development version. Python's dependency tracking is doing something that I don't understand in emerging dependencies of the older trepan3k with the newer one. 

> 
> The other work-around – downgrading `pytracer` to 0.3.2 – seems forbidden by `pip` (assuming `trepan3k` would even work with `pytracer<2.0.0`).
> 
> Installing from a [wheel](https://github.com/Trepan-Debuggers/python3-trepan/releases/download/1.3.0/trepan3k-1.3.0-39-none-any.whl) is not supported on my platform, so I did test that.
> 
> I am raising this issue in addition to [rocky/pytracer#8](https://github.com/rocky/pytracer/issues/8) because `trepan3k` is unusable in the current state, but also to ask @rocky to share the deeper understanding he seems to have of this error.

**Comment by nejmeijer** [comment]
While trying to reproduce the error in isolation I realised that this crash is specific to the Python version the debugging target is currently am stuck at (3.9.19). Is this version still within the scope of the `trepan3k` project? (If not, which other Python debugger would you recommend?)

The error can be reproduced with the following commands.
```
uname -a
mamba info
mamba create -n testenv python==3.9.19 pip
mamba activate testenv
pip install trepan3k
trepan3k
```
Here is the [log](https://github.com/user-attachments/files/18013373/trepan3k_crash.log) of the above on my machine.

**Comment by rocky** [comment]
Thanks. This is enough information to figure out what's wrong.

The 3.9 wheel on PyPI is missing that file `tracer.tracer.py`. I have removed the wheel from PyPI now.  If you want a 3.9 wheel, I just uploaded a corrected version to https://github.com/rocky/pytracer/releases/tag/2.0.0 . There is also an egg for 3.9.

The tarball  [tracer-2.0.0.tar.gz](https://files.pythonhosted.org/packages/de/a8/9f47e80c5fb2a908dd6f23f3eb22a182956425a75ca70f88be0384315361/tracer-2.0.0.tar.gz) on PyPI has the full contents, so presumably installing from that would work as well.

trepan3k and tracer and xdis, and ... are "supported" down to Python 3.0 or further. However, to get this working on older Python's  like in the 3.1 and before range you will need a modified pigments.
